cmd/root.go

```diff
@@ -4,33 +4,72 @@
 package cmd
 
 import (
+	"bytes"
 	"encoding/json"
+	"fmt"
 	"io"
 	"os"
-	"path/filepath"
 
 	"github.com/Azure/aks-engine/pkg/api"
 	"github.com/Azure/aks-engine/pkg/api/vlabs"
 	"github.com/Azure/aks-engine/pkg/armhelpers"
-	"github.com/Azure/aks-engine/pkg/helpers"
-	"github.com/Azure/go-autorest/autorest/azure"
+	"github.com/Azure/aks-engine/pkg/cli/config"
+	"github.com/bacongobbler/symdiff"
+	xdg "github.com/casimir/xdg-go"
+	"github.com/imdario/mergo"
 	"github.com/pkg/errors"
-	uuid "github.com/satori/go.uuid"
 	log "github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
 	flag "github.com/spf13/pflag"
-	ini "gopkg.in/ini.v1"
 )
 
 const (
+	envVarPrefix         = "AKS_ENGINE"
```

Comment:
Azure conventions dictate that `AZURE` be used as the prefix. This might fall out of the scope of this PR, but if this is a new convention we're introducing - we should reconsider.

Reply:
So this should be changed to AZURE_AKS_ENGINE?

Reply:
I meant that we would need to use AZURE as the envVar prefix for variables that have an Azure-wide scope and not within that of AKS_ENGINE. I think `AKS_ENGINE` as a prefix for aks-engine specific variables totally works.